Libraries/LibWeb/HTML/HTMLFieldSetElement.cpp

```diff
@@ -99,4 +100,11 @@ bool HTMLFieldSetElement::will_validate()
     return false;
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-validationmessage
+String HTMLFieldSetElement::validation_message() const
+{
+    // Returns the empty string, as fieldset elements are not subject to constraint validation.
+    return {};
+}
```

Comment:
I don't think this method is necessary. If this method wasn't here `FormAssociatedElement::validation_message()` should be called, which will return the empty string